README.md

```diff
@@ -100,24 +114,24 @@ client                         │                │
 disconnects                    │                │
 ```
 
-### Example: Client mock session
+#### Client mock session
```

Comment:
Called the whole section examples then The example wort can be removed from all the examples